decode.go

```diff
@@ -48,31 +48,43 @@ func parseLocalDate(b []byte) (LocalDate, error) {
 	if err != nil {
 		return date, err
 	}
+
 	date.Month = time.Month(v)
 
 	date.Day, err = parseDecimalDigits(b[8:10])
+	if err != nil {
+		return date, err
+	}
 
 	return date, nil
 }
 
+var ErrNotDigit = errors.New("not a digit")
+
 func parseDecimalDigits(b []byte) (int, error) {
 	v := 0
+
 	for _, c := range b {
 		if !isDigit(c) {
-			return 0, fmt.Errorf("expected digit")
+			return 0, fmt.Errorf("%s: %w", b, ErrNotDigit)
 		}
+
 		v *= 10
 		v += int(c - '0')
 	}
+
 	return v, nil
 }
 
+var ErrParseDateTimeMissingInfo = errors.New("date-time missing timezone information")
+
+const dateTimeByteLen = 6
```

Comment:
Cool with naming the const, but I'd rather see it defined close to where it is used.

Reply:
will do, maybe even within the function actually